src/Application.njs

```diff
@@ -16,11 +16,20 @@ class Application extends Nullstack {
     const { value } = await HelloPlugin.sayHello();
   }
 
+  retrieveSubscriptionTime() {
+    const monthlyChecked = document.getElementById("monthly").checked
```

Comment:
it would also be cool if we could have it (monthly) as a constant then it clearer wha this stands for